### PR TITLE
Add channel name in chaincode log in case of multi-channels

### PR DESCRIPTION
**Pull request by Kelvin-M**
## Description
In case we use the same chaincode for multiple channel, like https://github.com/SubstraFoundation/hlf-k8s/pull/102, we need to log the channel name for each request to be able to review logs in case of troubleshooting sessions.

## Closes issue(s)
None

## Companion PRs
https://github.com/SubstraFoundation/hlf-k8s/pull/102

## How to test / repro
Launch e2e tests (minimal) and look for chaincode logs :) 

## Screenshots / Trace
```

time="2020-11-25T10:39:38Z" level=info msg="Load TLS certificates"
time="2020-11-25T10:39:38Z" level=info msg="Start Substra ChaincodeServer"
time="2020-11-25T10:41:01Z" level=info msg="Args received by the chaincode on channel 'mychannel': []string{\"registerNode\", \"[\\\"\\\"]\"}"
time="2020-11-25T10:41:01Z" level=info msg="Response from chaincode on channel 'mychannel' (in 18ms): main.Node{ID:\"MyOrg1MSP\"}, error: %!s(<nil>)"
time="2020-11-25T10:41:02Z" level=info msg="Args received by the chaincode on channel 'yourchannel': []string{\"registerNode\", \"[\\\"\\\"]\"}"
time="2020-11-25T10:41:02Z" level=info msg="Response from chaincode on channel 'yourchannel' (in 23ms): main.Node{ID:\"MyOrg1MSP\"}, error: %!s(<nil>)"
time="2020-11-25T10:41:26Z" level=info msg="Args received by the chaincode on channel 'mychannel': []string{\"registerNode\", \"[\\\"\\\"]\"}"
time="2020-11-25T10:41:26Z" level=info msg="Response from chaincode on channel 'mychanne'l (in 7ms): main.Node{ID:\"MyOrg2MSP\"}, error: %!s(<nil>)"
time="2020-11-25T10:41:27Z" level=info msg="Args received by the chaincode on channel 'yourchannel': []string{\"registerNode\", \"[\\\"\\\"]\"}"
time="2020-11-25T10:41:27Z" level=info msg="Response from chaincode on channel 'yourchannel' (in 17ms): main.Node{ID:\"MyOrg2MSP\"}, error: %!s(<nil>)"

```
## Changes include
- [x] Add channel name in `Args received` log line
- [x] Add channel name in `Response from chaincode` log line

## Checklist
- [x] I have tested this code with e2e tests from substra-tests

## Other comments
Feel free to change the format if you want to log it in a clearer way !

### CODE REVIEW

File: chaincode/main.go [hunk 1]
```diff
@@ -229,6 +240,10 @@ func formatErrorResponse(err error) peer.Response {
 }
 
 func main() {
+	logger.SetFormatter(&logrus.TextFormatter{
+		ForceColors: true,
```
Comment (AurelienGasser) [inline chaincode/main.go:244]: :heart_eyes:

**Review by AurelienGasser — APPROVED** [review]
Nice addition

**Comment by AurelienGasser** [comment]
@Kelvin-M the last commit makes the code hard to read, maybe introduce a `mockTxid` variable and use it everywhere? Or better, set the tx id within `MockInvoke`

**Comment by AurelienGasser** [comment]
@Kelvin-M please take a look at my latest changes, it'f that's OK with you I think we can merge.

**Review by inelgnu — APPROVED** [review]
👍

**Pull request merged**